numba_dppy/utils/misc.py

```diff
@@ -32,3 +32,24 @@ def assert_no_return(rettype):
     if rettype is not None and rettype != types.void:
         msg = "DPPY kernel must have void return type but got {rettype}"
         raise TypeError(msg.format(rettype=rettype))
+
+
+class IndeterminateExecutionQueueError(Exception):
+    pass
+
+
+cfd_ctx_mgr_wrng_msg = (
```

Comment:
Can you move these into a module called `messages.py`

Reply:
Overall it is a good design, as it may someday help us support different languages easily. A translator will only need to update the strings in one place.